templates/templates/_navigation-developer-h.html

```diff
@@ -1,82 +1,230 @@
-          <a class="p-link--strong" href="https://tutorials.ubuntu.com" title="Visit the Ubuntu Tutorials - external site">Ubuntu Tutorials&nbsp;&rsaquo;</a>
-          <br>
-            Step-by-step tutorials for developers
-        </p>
-        <p class="p-p--small">
-          <a class="p-link--strong" href="https://docs.ubuntu.com" title="Visit the Ubuntu Docs - external site">Ubuntu Docs&nbsp;&rsaquo;</a>
-          <br>
-            The home of Ubuntu documentation
-        </p>
+        <h4 class="p-muted-heading--small">Support</h4>
       </div>
-      <div class="col-3">
-        <p class="p-p--small">
-          <a class="p-link--strong" href="https://blog.ubuntu.com" title="Visit the Ubuntu blog - external site">Ubuntu blog&nbsp;&rsaquo;</a>
-          <br>
-            Stay up to date with the latest Ubuntu news stories and resources
-        </p>
-        <p class="p-p--small">
-          <a class="p-link--strong" href="https://community.ubuntu.com/t/ask-ubuntu/21/1" title="Visit the Ask Ubuntu - external site">Ask Ubuntu&nbsp;&rsaquo;</a>
-          <br>
-            Share your technical know-how with others by answering questions
-        </p>
+      <div class="col-9">
+        <ul class="p-inline-list--middot is-x-dense">
+          <li class="p-inline-list__item">
+            <a href="/support/community-support">Community support</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://docs.ubuntu.com/">Read the docs</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://askubuntu.com/">Ask Ubuntu</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://wiki.ubuntu.com/IRC/ChannelList">IRC Channels</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://help.ubuntu.com/community/CommunityHelpWiki">Help</a>
+          </li>
+        </ul>
       </div>
+    </div>
+    <div class="row"><hr class="p-hr--subtle p-hr--dense"></div>
+    <div class="row">
       <div class="col-3">
-        <p class="p-p--small">
-          <a class="p-link--strong" href="/download">Download Ubuntu&nbsp;&rsaquo;</a>
-          <br>
-            Ubuntu is completely free to download, use and share
-        </p>
+        <h4 class="p-muted-heading--small">Security</h4>
+      </div>
+      <div class="col-9">
+        <ul class="p-inline-list--middot is-x-dense">
+          <li class="p-inline-list__item">
+            <a href="https://usn.ubuntu.com/">Security notices</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="/support/esm">Extended Security Maintenance</a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="/security">Security</a>
+          </li>
+        </ul>
       </div>
+    </div>
+    <div class="row"><hr class="p-hr--subtle p-hr--dense"></div>
+    <div class="row" style="padding-bottom: 1.5rem">
```

Comment:
Could you not use the `u-sv3`?

Reply:
I've added it. Can't add to the row because they both use `:after` so have wrapped it in another `div`